app/code/Magento/Contact/Api/Data/ContactFormInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Magento\Contact\Api\Data;
+
+interface ContactFormInterface
+{
+    const NAME = 'name';
```

Comment:
Because we prefer composition instead of abstraction we don't need to have these constants